inference-engine/src/inference_engine/include/openvino/runtime/infer_request.hpp

```diff
@@ -91,24 +91,6 @@ class INFERENCE_ENGINE_API_CLASS(InferRequest) {
      */
     std::vector<ProfilingInfo> get_profiling_info() const;
 
-    /**
-     * @brief Sets input data to infer
-     *
-     * @note Memory allocation doesn't happen
-     * @param inputs A reference to a map of input blobs accessed by input names.
-     *        The type of Blob must correspond to the network input precision and size.
-     */
-    void set_input(const std::map<std::string, std::shared_ptr<ie::Blob>>& inputs);
-
-    /**
-     * @brief Sets data that will contain result of the inference
-     *
-     * @note Memory allocation doesn't happen
-     * @param results - a reference to a map of result blobs accessed by output names.
-     *        The type of Blob must correspond to the network output precision and size.
-     */
-    void set_output(const std::map<std::string, std::shared_ptr<ie::Blob>>& results);
-
```

Comment:
let's remove `set_batch` as well

In new API we will support dynamism based on input data

Reply:
removed